lib/fizzy/parser.cpp

```diff
@@ -139,12 +139,18 @@ inline parser_result<Memory> parse(const uint8_t* pos, const uint8_t* end)
 
 inline parser_result<std::string> parse_string(const uint8_t* pos, const uint8_t* end)
```

Comment:
Actually I don't think this should be inline, as it is used by multiple methods.